src/test/java/edu/hm/hafner/analysis/IssueAssert.java

```diff
@@ -0,0 +1,193 @@
+package edu.hm.hafner.analysis;
+
+import org.assertj.core.api.AbstractAssert;
+
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * This class provides custom AssertJ assertion for {@link Issue}
+ *
+ * @author slausch
+ */
+public class IssueAssert extends AbstractAssert<IssueAssert,Issue> {
+
+    public IssueAssert(Issue actual){
+        super(actual,IssueAssert.class);
+    }
+
+    public static IssueAssert assertThat(Issue actual) {
+        return new IssueAssert(actual);
+    }
+
+    /**
+     * Checks whether an Issue has a specific filename.
+     * @param filename - String specifying filename.
+     * @return this
+     */
+    public IssueAssert hasFileName(String filename){
+        isNotNull();
+
+        if (!Objects.equals(actual.getFileName(), filename)) {
+            failWithMessage("Expected filename to be <%s> but was <%s>", filename, actual.getFileName());
+        }
+        return this;
+    }
+    /**
+     * Checks whether an Issue has a specific category.
+     * @param category - String specifying category.
+     * @return this
+     */
+    public IssueAssert hasCategory(String category){
+        isNotNull();
+
+        if (!Objects.equals(actual.getCategory(), category)) {
+            failWithMessage("Expected category to be <%s> but was <%s>", category, actual.getCategory());
+        }
+        return this;
+    }
+    /**
+     * Checks whether an Issue has a specific type.
+     * @param type - String specifying type.
+     * @return this
+     */
+    public IssueAssert hasType(String type){
+        isNotNull();
+
+        if(!Objects.equals(actual.getType(),type)) {
+            failWithMessage("Expected type to be <%s> but was <%s>", type, actual.getType());
+        }
+        return this;
+    }
+    /**
+     * Checks whether an Issue has a specific priority.
+     * @param priority - Priority specifying priority.
+     * @return this
+     */
+    public IssueAssert hasPriority(Priority priority){
+        isNotNull();
+
+        if (!Objects.equals(actual.getPriority(),priority)){
+            failWithMessage("Expected priority to be <%s> but was <%s>", priority, actual.getPriority());
```

Comment:
Der Standard ist:
`  private static final String EXPECTED_BUT_WAS_MESSAGE = "%nExpecting:%n <%s>%nto be equal to:%n <%s>%nbut was not.";
`